bin/pycbc_multi_inspiral

```diff
@@ -335,8 +335,13 @@ with ctx:
         'null_snr': float32,
         'nifo': int,
         'my_network_chisq': float32,
+        'my_network_bank_chisq': float32,
+        'my_network_cont_chisq': float32,
         'reweighted_snr': float32,
-        'slide_id': int
+        'slide_id': int,
+        'mass1': float32,
+        'mass2': float,
+        'mchirp': float
```

Comment:
float32

Reply:
See my comment above. Do not store the template parameters with the triggers. Look them up in the template bank itself, using the template ID or template hash provided with the triggers.